doc/JSON.md

```diff
@@ -0,0 +1,18 @@
+STEPS TO BUILD AND COMPILE THE PROJECT
+=======================================
+
+After Installing the prerequisites
+
+Step 1 : We first need to build the sail parser.
+        i) Find a directory to clone that repository, and make that directory your current working directory.
+       ii) `git clone -b json https://github.com/ThinkOpenly/sail`
```

Comment:
I made the "json" branch the default branch now, so that text is optional. Just FYI. You can remove that here or keep it -- both are correct.

Reply:
I think I'll remove the `-b json`, it will make cloning simpler.

Reply:
> I think I'll remove the `-b json`, it will make cloning simpler.

I don't see this change?

Reply:
Oops! I'll make the change right away!